doc/requirements.txt

```diff
@@ -6,6 +6,7 @@ myst-parser
 Sphinx==4.1.2
 sphinx-math-dollar==1.2
 sphinx-reredirects==0.0.1
+sphinx-rtd-theme
```

Comment:
The script seems to use `release/requirements.txt` (for unknown reasons).

Reply:
It's because I had made the `release/requirements.txt` file just before Travis got effectively disabled and we urgently needed to switch to Actions.

Reply:
I later on realized that the `doc/requirements.txt` was added quite recently. So the cause of the confusion is probably that rather than `release`.

Reply:
We need to migrate the CI away from the release requirements file, and also make the requirements less pinned and only list top-level packages https://github.com/sympy/sympy/issues/20623.

Reply:
In fact I don't know when the doc/requirements.txt was added, it also should not be using pins either, unless they are required (this is all of course completely unrelated to this PR).

Reply:
`doc/requirements.txt` was added in #22658. I'll remove the pinned version numbers in #22801, which is currently open.